gamemode/config/config.lua

```diff
@@ -500,3 +500,9 @@ Hungermod module
 GM.Config.hungerspeed = 2
 -- starverate <Amount> - How much health that is taken away every second the player is starving  (3 is the default).
 GM.Config.starverate = 3
+
+--[[---------------------------------------------------------------------------
+DO NOT TOUCH THIS (unless you want broken shit!)
+---------------------------------------------------------------------------]]
+-- Prepare minHitDistance for DistToSqr
+GM.Config.minHitDistance = GM.Config.minHitDistance ^ 2
```

Comment:
Presumably LuaJIT (or even the compiler?) would easily optimize a case as simple as `x^2`. In my opinion `x^2` conveys what you're trying to do better than `x*x` especially if `x` was a more complex variable name. To me this falls under category of premature optimization albeit it is a very small one.

Of course I come without benchmarks or any proof so happy to be proven wrong etc.